styles/components/_modalWindow.scss

```diff
@@ -0,0 +1,159 @@
+// Positioning /
+// Display & Box Model /
+// Color /
+// Text /
+// Other /
+
+.modal {
+    // Positioning /
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    z-index: 1;
+    // Display & Box Model /
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    // Color /
+    background-color: rgba(0,0,0,.8);
+    &__container {
+        // Display & Box Model /
+        height: 889px;
+        width: 1200px;
+        border-radius: 16px;
+        padding: 3rem 4rem;
+        display: flex;
+        flex-direction: column;
+        // Color /
+        background: #FFFFFF;
+        box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 4px 4px 40px rgba(255, 0, 0, 0.9);
+        // Text /
+        text-align: left;
+    }
+    &__heading {
+        margin: 0 auto;
+        text-align: center;
+        text-transform: uppercase;
+        line-height: 51px;
+    }
+    &__text {
+        margin: 1rem 0;
+        font-weight: 500;
+        font-size: 28px;
+        line-height: 34px;
+    }
+}
+
+.details {
+    margin-left: 285px;
+    height: 355px;
+    padding-top: .75rem;
```

Comment:
```suggestion
    padding: .75rem 0 0;
```

Reply:
Is it better to change it like that? And if other paddings are different?

Reply:
@vieraboschkova then you can overwrite them using padding-(...) declaration
https://codepen.io/redaxela/pen/rNMoKrJ 👀

Reply:
Yes, but I mean a case when I specifically defined paddings before, and now i just want to change one. This would reset the others, which would not be the desired outcome.